how-to/onboarding-guide.md

```diff
@@ -147,9 +148,13 @@ the `title:` yaml element and update `contributor_type:` as see below:
 
 * If they haven't already done, ask the new editor turn on [two-factor authentication (2FA) for GitHub](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa).
 
+* Give editors permissions they will need on GitHub to manage reviews:
+  * For ad-hoc guest editors: invite the editor to the `software-submission` repository
```

Comment:
I thought about adding screenshots like the ones you DM'd me @lwasser to show me how you were doing this -- with names changed to protect the innocent ofc. Not sure if that's overkill

Reply:
i think that's a great idea @NickleDave because if you asked - someone else will. AND see above i gave you maintainer access to the list. i also like that you distinguish adhoc guest editors from longer term editors. we just need to keep track of who is where. do we want to temporarily add a guest editor to the editor team, to our slack channel  and remove them there OR do we just add them to the repo is the question